.sync.yml

```diff
@@ -0,0 +1,52 @@
+---
+.travis.yml:
+  bundler_args: ' '
+  branches:
+    - release
+
+Gemfile:
+  required:
+    ':development':
+      - gem: 'nokogiri'
+        version: '1.8.4'
+      - gem: 'net-telnet'
+        version: '0.2.0'
```

Comment:
Does this module support Ruby 2.1.9? This version of net-telnet removes support for 2.1.9. I also believe this will conflict with the pin that the puppet-module-gems have of net-telnet at 0.1.1. So if you don't explicitly need a feature from 0.2.0, it's probably best to remove this and use the meta-gems pin.